cachito/workers/scm.py

```diff
@@ -209,6 +209,10 @@ def fetch_source(self, gitsubmodule=False):
                 OSError,  # raised by tarfile when an FS operation fails
             ) as exc:
                 log.warning("Error handling archived artifact '%s': %s", previous_archive, exc)
```

Comment:
(optional): Should we also log the exception type (or use repr)? Sometimes, the type says more than the message

Reply:
I added the repr to the msg as well. Not sure if the repr alone should be enough or if we better leave the string msg there as well

Reply:
Good point, we can keep both